src/main/java/org/example/persistence/utilities/AnnotationUtils.java

```diff
@@ -58,6 +67,15 @@ public static String getColumnName(Field field) {
         }
     }
 
+    public static String getColumnNameFromManyToOne(Field field) {
+        if (field.isAnnotationPresent(ManyToOne.class)) {
+            String fieldName = field.getAnnotation(ManyToOne.class).name();
+            return fieldName.equals("") ? field.getName() : fieldName;
```

Comment:
Consider this: in one case you will get return value "schoolClass", in the other case you will get "school_class_id".